src/script.js

```diff
@@ -200,6 +200,10 @@ export function getPartnerAttributionID() : ?string {
     return getSDKAttribute(SDK_SETTINGS.PARTNER_ATTRIBUTION_ID);
 }
 
+export function getMerchantRequestedPopupDisabled() : boolean {
```

Comment:
I wonder if we need this new function or if checkout-components can just use `getSDKAttribute()` directly instead.

Reply:
This is added mainly for consistency to follow the current pattern of accessing all attributes via a helper function.